src/lib/components/SegmentButton.svelte

```diff
@@ -53,7 +58,14 @@
     padding: var(--padding-0_5x) var(--padding-1_5x);
 
     @include text.truncate;
-    transition: color var(--animation-time-normal);
+    transition:
+      color var(--animation-time-normal),
```

Comment:
Does color need to be animated if opacity is already animated?
And why to they have a different animation time?

Reply:
Color for selection, opacity for initial animation.